types/office-js-preview/index.d.ts

```diff
@@ -12616,7 +12616,7 @@ declare namespace Office {
          */
```

Comment:
consider also updating the attachmentType description: Currently says "Gets a value that indicates the type of an attachment." Could say "Gets a value that indicates the type of the attachment." or "Gets a value that indicates the attachment's type." Here and in office-js version